### DISCUSSION TITLE
How to use the UI with no login, but read-only mode ?

### DISCUSSION THREAD

**Discussion by dgnemo** [argoproj/argo-cd#12284] [Q&A] [answered]
Hi!

In my current setup we want to leave the ArgoCD UI open for everybody (no login?), but in read-only mode.

Currently I disabled Argo CD RBAC for user authentication (via `configs.params."server.disable.auth"`) but I could not find how to limit the UI to ready-only.

Any Idea how I can do it?
Thanks!

**Comment by crenshaw-dev** [comment, marked as answer]
Don't completely disable auth, because that disables both authentication _and_ authorization (i.e. anyone can do anything).

Instead, use anonymous mode with a default policy granting read-only permissions: https://argo-cd.readthedocs.io/en/latest/operator-manual/rbac/#anonymous-access

**Reply by dgnemo** [reply]
Thanks Michael! it worked perfectly.

**A note for users of the [ArgoCD Helm Chart](https://github.com/argoproj/argo-helm/tree/main/charts/argo-cd):**  
the documentation does lists `configs.cm."admin.enabled" : true/false`,  
and `configs.rbac."policy.default` (which you want to set to `role:readonly`)
but you also have to add `configs.cm.users."anonymous.enabled": true/false` (which is not documented in the Helm Chart repo).

**Comment by dgnemo** [comment]
Thank you Michael!!
Will try it shortly

Do you know if the default read-only policy allows the (anonymous) users to
request a refresh the apps ?

I would like the users to have that option

**Reply by crenshaw-dev** [reply]
It should work! Users only need `get` access to trigger a refresh.
